test/unit/controller.js

```diff
@@ -894,7 +905,9 @@ exports['controller.root'] = {
 
     this.child = new FakeChild();
 
-    this.spawn = this.sandbox.stub(cp, 'spawn', () => this.child);
+    this.spawn = this.sandbox.stub(cp, 'spawn', () => {
+      return this.child;
+    });
```

Comment:
No, but it's possible that `jsbeautify` did that. You should've run it anyway :P